features/steps/steps.py

```diff
@@ -13,19 +13,12 @@
 )
 
 from features.environment import create_uat_image
-from features.util import (
-    SLOW_CMDS,
-    emit_spinner_on_travis,
-    launch_lxd_container,
-    lxc_exec,
-    nullcontext,
-    wait_for_boot,
-)
+from features.util import SLOW_CMDS, emit_spinner_on_travis, nullcontext
 
 from uaclient.defaults import DEFAULT_CONFIG_FILE
 
 
-CONTAINER_PREFIX = "behave-test-"
+CONTAINER_PREFIX = "ubuntu-behave-test-"
```

Comment:
Why change this?

Reply:
This is just to guarantee that the instance name will be properly parsed by this regex:
https://github.com/canonical/pycloudlib/blob/master/pycloudlib/lxd/cloud.py#L580